frontend/src/utils/defaultChangesetComment.js

```diff
@@ -0,0 +1,8 @@
+export const retrieveDefaultChangesetComment = (changesetComment, projectId) => {
+  let pattern = `\\w*(-project-${projectId})$`;
```

Comment:
Here we could use the `TM_DEFAULT_CHANGESET_COMMENT` [variable](https://github.com/hotosm/tasking-manager/blob/develop/example.env#L105) set in config file

Reply:
Just realized that this  variable is accessible only in the backend. We should let this information accessible through the frontend config settings, so that the default comment update in one place reflects across the application